source/manuals/monitor-paas-app-with-prometheus.html.md.erb

```diff
@@ -37,7 +37,43 @@ Create a Prometheus service within your PaaS space and allow it to bind to apps
   - `cf update-service <cdn-route-service> \
     -c '{"domain": "<custom-domain>", "headers": ["Accept", "Authorization", "<other app header values>"]}'`
 
-Within 5 minutes Prometheus will start scraping your application for metrics, you can validate this by checking [Grafana][].
+Within 10 minutes Prometheus will start scraping your application for metrics, you can validate this by checking [Grafana][].
+
+### When using zero downtime plugins or a `blue-green` deployment process
+
+#### IP Whitelist your applications metrics endpoint
+
+If you're using a [blue-green deployment process][] with a zero downtime plugin such as [autopilot][] you should disable the basic auth on the metrics endpoint when using the [Ruby gem][] or [Python library][] and instead protect the metrics endpoint using IP whitelisting in order to minimise gaps in metrics between deployments.
+
+By using the `re-ip-whitelist-service` you will only allow traffic from GDS Prometheis and [GDS Office IPs][].
+
+1. Register the IP whitelist route service as a user-provided service in your PaaS space.
+
+    `cf create-user-provided-service re-ip-whitelist-service -r https://re-ip-whitelist-service.cloudapps.digital`
+
+2. Map the route to the metrics path.
+
+  - You can do this either manually:
+
+    `cf map-route app-to-protect cloudapps.digital --hostname app-to-protect --path metrics`
+
+  - Or by adding a route to your routes section in your `manifest.yml`:
+
+```
+      routes:
+      ...
+      - route: app-to-protect.cloudapps.digital/metrics
+```
+
+3. Register the IP whitelist route service against the metrics path.
```

Comment:
I assume there is no corresponding action to go in a manifest for this one and they only need to do this once the very first time they deploy?

If so, maybe it would work to swap the order of 1 and 2 so the thing that needs the change in the manifest comes first and then the two individual actions which don't rely on a manifest come together?

Reply:
Thinking about it, I'm going to change the text again as it should really just be in the `manifest` I think and not manually set